examples/tutorial/tutorial_notebooks/introduction/exercise_4.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
This file should be renamed exercise_5 since we now have a new exercise_4 in `introduction/`